clang/test/dpct/helper_files_ref/include/lib_common_utils.hpp

```diff
@@ -49,7 +49,7 @@ inline void mkl_get_version(version_field field, int *result) {
   }
 }
 
-enum class library_data_t : unsigned char {
+enum library_data_t {
```

Comment:
Why we make this change and use unscoped enumerations?

Reply:
ncclDataType_t is a enum type, not enum class.
In some code, the value of enum may be used directly.
But enum class cannot be converted implicitly

Reply:
Ah, maybe we can move unscoped enumerations into a namespace, but not export these enumerations symbols to dpct namespace directly
···
namespace class_library_data_t {
enum {
 ....
};
} // namespace class_library_data_t
···